examples/misc/bin/try_dart/functions.dart

```diff
@@ -0,0 +1,22 @@
+// A function declaration.
+int timesTwo(int x) {
+  return x * 2;
+}
+
+// Arrow syntax is shorthand for `{ return expr; }`.
+int timesFour(int x) => timesTwo(timesTwo(x));
+
+// Functions are objects.
+int runTwice(int x, Function f) {
```

Comment:
Why did this change from `int Function(int) f`?

Reply:
It currently is `Function f` on dart.dev. I changed it to `int Function(int) f` in the first commit to this PR, and then reverted the change so that we could separately discuss and address any required code changes in another PR.

Reply:
ok, no problem.